.github/workflows/github-action-scan.yml

```diff
@@ -3,27 +3,17 @@ name: Build SecHub GHA (scan)
 
 on:
   push:
-    branches:
-      - main
-      - master
-      - develop
-      - hotfix
+    paths:
+      - 'github-actions/**'
+      - '.github/workflows/github-action*'
```

Comment:
why should this workflow run if changes to other workflow files are made?

I think you should just remove the wildcard and set the full name instead

Reply:
full path and name added